v3/src/components/case-table/case-table.scss

```diff
@@ -323,6 +323,13 @@ $table-body-font-size: 8pt;
         background-color: var(--rdg-row-selected-background-color);
       }
 
+      &.right-justify {
+        justify-content: flex-end;
+        // font-family: 'Roboto  Mono', monospace;
+        font-family: 'Inconsolata', monospace;
```

Comment:
I told myself I wasn't going to do this, but I couldn't resist looking into other fonts. Take a look at it after my commit and see what you think.